consensus/core/src/tests/universal_committer_tests.rs

```diff
@@ -57,6 +58,272 @@ fn direct_commit() {
     };
 }
 
+/// Ensure idempotent replies.
+#[test]
+fn idempotence() {
+    telemetry_subscribers::init_for_testing();
+    // Commitee of 4 with even stake
+    let context = Arc::new(Context::new_for_test(4).0);
+    let dag_state = Arc::new(RwLock::new(DagState::new(
+        context.clone(),
+        Arc::new(MemStore::new()),
+    )));
+
+    // Create committer without pipelining and only 1 leader per round
```

Comment:
1 leader per round or per wave?

Reply:
spoke offline, but 1 leader per leader round to capture that multi leader is disabled.